_posts/2024-11-12-behind-the-code-msub2.md

```diff
@@ -0,0 +1,53 @@
+---
+layout:     post
+tags:       blog
+title:      "Behind the code: an interview with msub2"
+date:       2024-11-12
+summary:    Q&A with Servo contributor msub2.
+categories:
+---
+
+Behind the Code is a new series of interviews with the contributors who help propel Servo forward. Ever wondered why people
+choose to work on web browsers, or how they get started? We invite you to look beyond the project's pull requests and issue
+reports, and get to know the humans who make it happen.
+
+TODO: add some links to neat contributions
+
+# <img src="https://avatars.githubusercontent.com/u/70986246?v=4" width=60> [msub2](https://github.com/msub2)
```

Comment:
Mostly a nit, but the post title is already an `<h1>`, so let's use smaller headers here.

```suggestion
## <img src="https://avatars.githubusercontent.com/u/70986246?v=4" width=60> [msub2](https://github.com/msub2)
```